template/Goerli-DssSpell.sol

```diff
@@ -40,17 +42,20 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
     //
     // A table of rates can be found at
-    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
+    //    https://ipfs.io/ipfs/QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn
```

Comment:
Why is this one changing?

Reply:
I've applied the change in the collater one as well.
It updates with latest IPFS Rates Table (including also 0.01%)